apps/src/studio/studio.js

```diff
@@ -1066,6 +1066,8 @@ Studio.onTick = function () {
     Studio.executeQueue('when-right');
     Studio.executeQueue('when-down');
 
+    Studio.executeQueue('askCallback');
```

Comment:
given the complexity of this execution flow, it's probably worth a comment here explaining the purpose of this queue